### PR TITLE
Added option to decode to RGB instead of RGBA

### PR DESCRIPTION
**Pull request by Sunstradamus**
This change is also backwards compatible to <0.3.5. I'm not quite sure what's meant by an internal pull request in the contrib guidelines?

### CODE REVIEW

**Comment by Sunstradamus** [comment]
The default behavior of decoding as RGBA has lead to soft errors when it comes to integrating with other libraries (e.g. Three.JS) that expect the decoded data to be in RGB as jpegs don't contain alpha channel data.

**Comment by j-o-d-o** [comment]
Quite an old PR. Any chance this will be merged at some point? Would be a good feature.

**Comment by patrickhulce** [comment]
Yeah this LGTM to land if we can just get a test in here for it :)

**Comment by Sunstradamus** [comment]
Added a test case. By the way, there's quite a few unused images in `test/fixtures` and at least one is mislabeled (rgb.rgb should be rgb.rgba) - is there any reason to keeping them around? Removing them won't break any existing tests.

File: test/index.js [hunk 1]
```diff
@@ -213,3 +213,14 @@ it('should be able to decode a JPEG with options', function(t) {
   t.assert(rawImageData.data instanceof Uint8Array, 'data is a typed array');
   t.end();
 });
+
+it('should be able to decode a JPEG into RGB', function(t) {
+  var jpegData = fixture('grumpycat.jpg');
+  var rawImageData = jpeg.decode(new Uint8Array(jpegData), { useTArray: true, formatAsRGBA: false });
+  t.equal(rawImageData.width, 320);
+  t.equal(rawImageData.height, 180);
```
Comment (patrickhulce) [inline test/index.js]:
```suggestion
  t.equal(rawImageData.height, 180);
  t.equal(rawImageData.data.length, 320 * 180 * 3);
```

just to be extra sure :)

**Comment by patrickhulce** [comment]
> is there any reason to keeping them around?

not really, they might have been useful at some point but we can always checkout an older version if we need to revive them for some reason

**Pull request merged**